spec/support/decision_reviews/with_4142_2024.json

```diff
@@ -0,0 +1,275 @@
+        "separationPay": {
+          "received": true,
+          "payment": {
+            "serviceBranch": "Air Force"
+          },
+          "receivedDate": {
+            "year": "2000"
+          }
+        }
+      },
+      "serviceInformation": {
+        "servicePeriods": [
+          {
+            "serviceBranch": "Air Force",
+            "activeDutyBeginDate": "1980-02-05",
+            "activeDutyEndDate": "1990-01-02"
+          },
+          {
+            "serviceBranch": "Air Force Reserves",
+            "activeDutyBeginDate": "1990-04-05",
+            "activeDutyEndDate": "1999-01-01"
+          }
+        ],
+        "confinements": [
+          {
+            "confinementBeginDate": "1987-02-01",
+            "confinementEndDate": "1989-01-01"
+          },
+          {
+            "confinementBeginDate": "1990-03-06",
+            "confinementEndDate": "1999-01-01"
+          }
+        ],
+        "reservesNationalGuardService": {
+          "title10Activation": {
+            "anticipatedSeparationDate": "2020-01-01",
+            "title10ActivationDate": "1999-03-04"
+          },
+          "obligationTermOfServiceFromDate": "2000-01-04",
+          "obligationTermOfServiceToDate": "2004-01-04",
+          "unitName": "Seal Team Six",
+          "unitPhone": {
+            "areaCode": "123",
+            "phoneNumber": "1231231"
+          },
+          "receivingInactiveDutyTrainingPay": true
+        },
+        "alternateNames": [
+          {
+            "firstName": "JKack",
+            "middleName": "Clint",
+            "lastName": "Bauer"
+          }
+        ]
+      },
+      "treatments": [
+        {
+          "startDate": "2018-03-02",
+          "endDate": "2018-03-03",
+          "treatedDisabilityNames": [
+            "PTSD (post traumatic stress disorder)"
+          ],
+          "center": {
+            "name": "Private Facility 2",
+            "country": "USA"
+          }
+        },
+        {
+          "startDate": "2015-04-03",
+          "endDate": "2018-03-04",
+          "treatedDisabilityNames": [
+            "PTSD personal trauma"
+          ],
+          "center": {
+            "name": "Huntsville VA Facility",
+            "country": "USA"
+          }
+        }
+      ],
+      "disabilities": [
+        {
+          "diagnosticCode": 9999,
+          "disabilityActionType": "NEW",
+          "name": "PTSD (post traumatic stress disorder)",
+          "ratedDisabilityId": "1100583",
+          "secondaryDisabilities": [
+            {
+              "name": "PTSD personal trauma",
+              "disabilityActionType": "SECONDARY",
+              "serviceRelevance": "Caused by a service-connected disability\nLengthy description"
+            }
+          ]
+        }
+      ]
+    }
+  },
+  "form526_uploads": null,
+  "form4142": {
+    "privacyAgreementAccepted": true,
+    "limitedConsent": "Test data for limiting consent",
+    "providerFacility": [
+      {
```

Comment:
Would this hash also still include the initial "treatedDisabilityNames" hash that informed the addition of the "conditionsTreated" key, or would that only be in the post-transform json file?

Reply:
I suppose I thought "treatedDisabilityNames" was the DB-stored value, and "conditionsTreated" was what was later added to the hash.

Reply:
I thought that because the "conditionsTreated" key wasn't added to each provider facility until it was passed through the processor, that this representation "coming from the DB" would ONLY have "treatedDisabilityNames" as a key after it was processed.